modules/serverless-pingsource-ODC.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-pingsource.adoc
+
+[id="serverless-pingsource-ODC_{context}"]
+= Creating a PingSource using the Developer Perspective
+
+The following sections describe how to create and verify a basic PingSource from the {product-title} web console.
```

Comment:
You can create and verify a basic PingSource from the {product-title} web console.